### PR TITLE
Done with Checkout page

### PR DESCRIPTION
**Pull request by Monilprajapati**
Done with Checkout Page just one back button is remaining for components  will add it in upcoming commits

### CODE REVIEW

File: client/src/components/Payment/Payment.jsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                className="mt-3 w-4 h-4"
+                value={"card"}
+                name="payment"
+                checked
+              />) : (<input
+                type="radio"
+                className="mt-3 w-4 h-4"
+                value={"card"}
+                name="payment"
+              />)
```
Comment (victorchrollo14) [inline client/src/components/Payment/Payment.jsx:31]: you can use a single radio button over here, you can use checked = {true or false} to select and unselect a radio button.
you can do something like `checked = {paymentMethod === "card"}`. 
* (suggestion) you are using a lot of radiobuttons, you could have made a reusable component out of it.

**Comment by victorchrollo14** [comment]
Also put all the checkout specific components under checkout/ itself.

**Review by victorchrollo14 — CHANGES_REQUESTED** [review]

File: client/src/components/Payment/Payment.jsx (hunk 1 above)
Comment (Monilprajapati) [reply client/src/components/Payment/Payment.jsx]: Done

File: client/src/components/Payment/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { useState } from "react";
+
+const Card = () => {
+  const [card, setCard] = useState({})
+
+  const cardDetails = () => {
+    setCard({ ...card, [e.target.name]: e.target.value })
+  }
```
[the remark this answers is not in the document]
Comment (Monilprajapati) [reply client/src/components/Payment/Card.jsx]: Done

File: client/src/components/Payment/Payment.jsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+            <div className="rezorpay pt-3">
+              <svg
+                width="143"
+                height="31"
+                viewBox="0 0 143 31"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <g id="razorpay" clip-path="url(#clip0_447_1303)">
+                  <path
```
[the remark this answers is not in the document]
Comment (Monilprajapati) [reply client/src/components/Payment/Payment.jsx]: Done

File: client/src/components/Payment/Payment.jsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                      height="30"
+                      fill="white"
+                      transform="translate(0.922852 0.5)"
+                    />
+                  </clipPath>
+                </defs>
+              </svg>
+            </div>
+          </div>
+        </div>
```
[the remark this answers is not in the document]
Comment (Monilprajapati) [reply client/src/components/Payment/Payment.jsx]: Done

File: client/src/components/Payment/Card.jsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+          onChange={(e) => cardDetails(e)}
+        />
+      </div>
+      <div className="countryname flex flex-col gap-1">
+        <label htmlFor="countryname">Country</label>
+        <input
+          type="text"
+          name="country"
+          className="bg-white px-2 py-2 rounded-md border border-white outline-none"
+          onChange={(e) => cardDetails(e)}
```
[the remark this answers is not in the document]
Comment (Monilprajapati) [reply client/src/components/Payment/Card.jsx]: Done

File: client/src/components/Shipping.jsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  return (
+    <>
+      <div className="name flex flex-col gap-1">
+        <label htmlFor="name">Your Name</label>
+        <input
+          type="text"
+          name="name"
+          className="bg-white px-2 py-2 rounded-md border border-white outline-none"
+          onChange={(e) => shippingDetails(e)}
+        />
```
[the remark this answers is not in the document]
Comment (Monilprajapati) [reply client/src/components/Shipping.jsx]: Done

**Comment by victorchrollo14** [comment]
ok looks good. Nice work

**Pull request merged**